Resources/Private/Language/PluginHaiku/de.locallang.xlf

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
```

Comment:
Shall we make a German translation file here? Wouldn't the best practicce way be to show a crowdin integration?

Reply:
Is it in our scope to demonstrate a Crowdin integration? Then this should be handled in another PR. Before that we have to streamline the xlf files accordingly (resname, message), probably use version 1.2 of XLF (which is on my todo).

I added it manually, so that one can really check if the language is used (and for me to check if it works).

Reply:
We actually describe in TYPO3 Explained how to use Crowdin, but I haven't looked into it yet. https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Internationalization/TranslationServer/Crowdin/ExtensionIntegration.html I agree to do this in another PR